lib/beacon/authorization/behavior.ex

```diff
@@ -0,0 +1,20 @@
+defmodule Beacon.Authorization.Behaviour do
```

Comment:
I feel like putting the word "Behaviour" in the module name is not generally accepted. Here are some other names for brainstorming.

```
Beacon.Authorization
Beacon.Authorization.Policy
Beacon.Authorization.Rules
```

Reply:
I believe @xn was following the existing pattern where we have `Beacon.DataSource.Behaviour` but I agree with this change, it makes more sense. We can even rename `Beacon.DataSource.Behaviour` in a following PR, we're okay with making breaking changes while it's still pre-release.

Reply:
I feel you. I was following the pattern set up with `Beacon.DataSource.Behaviour`

If we want to break that pattern, I'd say we should do it in a separate PR and unify the both of them.